packages/select/src/vaadin-lit-select.js

```diff
@@ -47,6 +47,11 @@ class Select extends SelectBaseMixin(ElementMixin(ThemableMixin(PolylitMixin(Lit
           flex: 1;
         }
 
+        :host(:not([focus-ring], [disabled])) [part='input-field'] {
```

Comment:
Any reason to not prevent it in the disabled state too?

Reply:
Yeah, good point. I don't remember why I added that. I think I had some reason, copied from another component. I’ll take a look again.

Reply:
It’s because otherwise the disabled field will get an unwanted border:
![Screenshot 2025-05-19 at 10 12 48](https://github.com/user-attachments/assets/ba6dec17-eea8-4b4c-bf41-df49aab31731)

I need to split the properties in two separate selectors.

Reply:
Okay, I think I managed to satisfy all states now.